### DISCUSSION TITLE
Download offline wallpaper

### DISCUSSION THREAD

**Discussion by liuyuzhou-it** [zixpo/candybar#148] [Q&A]
I have an offline wallpaper built into the icon pack, but it doesn't seem to download to the phone.

**Comment by sarsamurmu** [comment]
Let me check.

**Comment by sarsamurmu** [comment]
The fix will be in the next update. If you want that fix right now then use `9e5f974e8c` as the version (instead of `3.17.5`)

**Reply by liuyuzhou-it** [reply]
Thank you very much for your help to solve the problem I have been bothering for a long time.

**Reply by liuyuzhou-it** [reply]
The wallpaper downloaded to your phone seems to be wrong.

**Reply by sarsamurmu** [reply]
Can you explain?

**Reply by liuyuzhou-it** [reply]
An invalid wallpaper was downloaded, not the original offline wallpaper.

**Reply by sarsamurmu** [reply]
Use the newly released version
